pkg/retry/retry.go

```diff
@@ -201,18 +202,21 @@ func (s *Server) handleRetry() http.Handler {
 				}
 			}
 
+			logger.Infow("detected a failed event and successfully redelivered", "eventID", eventIdentifier.eventID)
```

Comment:
too much logging noise, moving this down so we log only when we successfully redeliver a failure after the fact.